sktime/forecasting/compose/_hierarchy_ensemble.py

```diff
@@ -159,6 +159,23 @@ def _aggregate(self, y):
 
         return Aggregator().fit_transform(y)
 
+    @property
+    def fitted_list(self):
+        """Deprecated attribute.
+
+        TODO 0.37.0: Remove this property entirely.
+        """
+        import warnings
+
+        warnings.warn(
+            """The fitted_list property is deprecated
```

Comment:
Kindly mention the name of the forecaster, so the user knows which component it is coming from